backend/fms_core/models/dataset.py

```diff
@@ -31,17 +31,29 @@ def validation_status(self) -> Optional[ValidationStatus]:
         if readset:
             return readset.validation_status
 
+    @property
+    def validated_by(self):
+        readset = self.readsets.first()
+        if readset.validated_by:
+            return readset.validated_by.username
+
+    @property
+    def released_by(self):
+        readset = self.readsets.first()
+        if readset.released_by:
+            return readset.released_by.username
```

Comment:
Risky thing if we have existing datasets that are partially released

Reply:
released_by and validated_by are used to indicate progression in the processing of the dataset more than indicate the detail portrait ... These essentially only there to report who did the release of the dataset content (be it release or blocking).

Reply:
in this case i propose to change all of these following properties, found in the model/dataset.py 


    @property
    def validation_status(self) -> Optional[ValidationStatus]:
        readset = self.readsets.first()
        if readset:
            return readset.validation_status

    @property
    def validated_by(self):
        readset = self.readsets.first()
        if readset.validated_by:
            return readset.validated_by.username

    @property
    def released_by(self):
        readset = self.readsets.first()
        if readset.released_by:
            return readset.released_by.username
            
since i used the already existing validation status property logic